src/Extension/CoreExtension.php

```diff
@@ -296,7 +296,7 @@ public function getOperators(): array
     {
         return [
             [
-                'not' => ['precedence' => 50, 'class' => NotUnary::class],
+                'not' => ['precedence' => 50, 'future_precedence' => 70, 'class' => NotUnary::class],
```

Comment:
Please update the phpdoc type in ExtensionInterface to document the optional `future_precedence` key here, as the implementation actually makes it possible for third-party extensions to use it for their own operators.

Reply:
Done